inference-engine/src/mkldnn_plugin/nodes/mkldnn_mvn_node.cpp

```diff
@@ -850,6 +819,7 @@ void MKLDNNMVNNode::createPrimitive() {
     jcp.normalize_variance = normalizeVariance_;
     jcp.across_channels = acrossChannels_;
     SizeVector in_dims = getParentEdgeAt(0)->getDims().ToSizeVector();
+    transformTo5DCase(in_dims);
```

Comment:
jcp.across_channels is used in kernels, and should be the transformed value, so we should move this before creating jcp.

Reply:
@chenhu-wang @a-sidorova I am wondering why we have green CI in that case? Please consider test coverage extension

Reply:
some more explanations:
"common subgraph pattern - mvn + mul + add (gamma * mvn + beta)" passed because a coincidence. 
[Here](https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/src/mkldnn_plugin/nodes/mkldnn_mvn_node.cpp#L77) and [here](https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/src/mkldnn_plugin/nodes/mkldnn_mvn_node.cpp#L400)
```
if (jcp_.planar_layout || jcp_.across_channels) {
    uni_vbroadcastss(vmm_mean, ptr[reg_mean]);
 } 
```
across_channels and planar_layout together checking enforce the right code path(1[2]d must be planar). If code is written in another way or if across_channels used in other place, then will lead to bug. For robust code, I would suggest move a little bit prior.

Reply:
@chenhu-wang thanks!

@dmitry-gorokhov there are two conditions with `jcp.across_channels` in kernels: only in code space for blocked or npsc layouts or in conditions `if (jcp.is_planar || jcp.across_channels)`. We know that `across_channels` might will be changed only for 1D or 2D cases, but 1D and 2D shapes are only planar layouts. So we won't go into code space for blocked or npsc layouts and we will get true in conditions `if (jcp.is_planar || jcp.across_channels)` anyway.
So we cannot cover these cases by tests